tartufo/scanner.py

```diff
@@ -68,12 +68,12 @@ def __init__(
         self.chunk = chunk
         self.logger = logging.getLogger(__name__)
 
-    def as_dict(self, compact=False) -> Dict[str, Optional[str]]:
+    def as_dict(self, output_format=None) -> Dict[str, Optional[str]]:
```

Comment:
I think it makes the most sense to keep this method the way it was, using a boolean. Since the other values of `output_format` mean nothing to this, a boolean seems cleanest.